### ISSUE TITLE
dependency problems lead to failed install on Ubuntu 22.10 kinetic 

### ISSUE THREAD

**Issue by FilLTP89** [qgis/QGIS#50840]
### What is the bug or the crash?

Hi
when installing QGIS on Ubuntu 22.10 kinetic, I get the following errors:
```
/usr/lib/qgis/crssync: error while loading shared libraries: libarmadillo.so.10: cannot open shared object file: No such file or directory

dpkg: error processing package qgis-providers (--configure):
 installed qgis-providers package post-installation script subprocess returned error exit status 127
Setting up libqt53dextras5:amd64 (5.15.6+dfsg-1) ...
dpkg: dependency problems prevent configuration of qgis:
 qgis depends on qgis-providers (= 1:3.28.0+git20221108+0e0a3eda90+37kinetic); however:
  Package qgis-providers is not configured yet.

dpkg: error processing package qgis (--configure):
 dependency problems - leaving unconfigured
Setting up libqgis-customwidgets (1:3.28.0+git20221108+0e0a3eda90+37kinetic) ...
No apport report written because the error message indicates its a followup error from a previous failure.
                                                                                                         
dpkg: dependency problems prevent configuration of qgis-plugin-grass:
 qgis-plugin-grass depends on qgis (= 1:3.28.0+git20221108+0e0a3eda90+37kinetic); however:
  Package qgis is not configured yet.

dpkg: error processing package qgis-plugin-grass (--configure):
 dependency problems - leaving unconfigured
Setting up qt3d-gltfsceneio-plugin:amd64 (5.15.6+dfsg-1) ...
No apport report written because the error message indicates its a followup error from a previous failure.

dpkg: dependency problems prevent configuration of python3-qgis:
 python3-qgis depends on qgis-providers (= 1:3.28.0+git20221108+0e0a3eda90+37kinetic); however:
  Package qgis-providers is not configured yet.

dpkg: error processing package python3-qgis (--configure):
 dependency problems - leaving unconfigured
Setting up python3-nbformat (5.4.0-2) ...
No apport report written because MaxReports is reached already
Setting up qt3d-assimpsceneimport-plugin:amd64 (5.15.6+dfsg-1) ...
```

I've tried different repositories in my ```/etc/apt/sources.list.d/qgis.sources``` file:

- ```https://ubuntu.qgis.org/ubuntu```
- ```https://ubuntu.qgis.org/ubuntu-nightly-release```
- ```https://ubuntu.qgis.org/ubuntu-ltr```
- ```https://ubuntu.qgis.org/ubuntu-nightly-ltr```
- ... 

An exemple of my ```/etc/apt/sources.list.d/qgis.sources``` file:
```
Types: deb deb-src
URIs: https://ubuntu.qgis.org/ubuntu-nightly
Suites: kinetic
Architectures: amd64
Components: main
Signed-By: /etc/apt/keyrings/qgis-archive-keyring.gpg
```

I still cannot find any satisfactory solution: the install fails whatsoever. Installing ```libarmadillo```  did not change the error.

Thanks!

### Steps to reproduce the issue

Follow the installation instructions, with file ```/etc/apt/sources.list.d/qgis.sources``` that reads:

```
Types: deb deb-src
URIs: https://ubuntu.qgis.org/ubuntu-nightly
Suites: kinetic
Architectures: amd64
Components: main
Signed-By: /etc/apt/keyrings/qgis-archive-keyring.gpg
```
and then:
```sudo apt-get install qgis qgis-plugin-grass -y```

### Versions

Tested with:

- 3.28.x Firenze RC
- 3.22.x Białowieża LTR
- 3.29 master

### Supported QGIS version

- [x] I'm running a supported QGIS version according to the roadmap.

### New profile

- [ ] I tried with a new QGIS profile

**Comment by Jack-Whattam** [comment]
I upgraded to Ubuntu 22.10 last night and installed QGIS 3.22x Białowieża via the ppa (https://launchpad.net/~ubuntugis/+archive/ubuntu/ppa) as per the last comment in this thread (https://www.reddit.com/r/QGIS/comments/yqq50v/installing_on_ubuntu_2004_no/). No dependency issues here.

I haven't tested it in detail yet but it is at least functional for opening and doing some basic shapefile editing with projects cretated in the same version of QGIS in Ubuntu 22.04.

**Comment by nicogodet** [comment]
As explained in Note in https://www.qgis.org/en/site/forusers/alldownloads.html#repositories, dependencies can be broken in unstable channel.

**Comment by FilLTP89** [comment]
Hello
just a quick update: after several, several attempts, I found a suitable workaround (waiting for ubuntu's stable release).

I've used the floowing repository: [https://qgis.org/ubuntu-nightly](https://qgis.org/ubuntu-nightly) 
which effectively contains source code and binary `adm64` for ubuntu 22.10 `kinetic` release.

Before installing `qgis` and `qgis-plugin-grass` I've installed `qgis-providers-common`, `qgis-providers`.

Then, I've installed the missing dependencies (from previous ubuntu distributions):

`sudo dpkg -i libarmadillo10_10.8.2+dfsg-1_amd64.deb`

`sudo dpkg -i libpoppler118_22.02.0-2ubuntu0.1_amd64.deb`

`sudo dpkg -i libproj22_8.2.1-1_amd64.deb`

Now it works!
